frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -97,6 +99,12 @@ pub fn expand_outer_config(
 				Ok(())
 			}
 		}
+
+		impl #scrate::traits::RuntimeGenesisBuild for RuntimeGenesisConfig {
```

Comment:
It seems to me that instead of a whole new trait, we can simply do this by `impl GenesisBuild for Tuple`.

Reply:
Could elaborate more/refer to some example on this idea?

Reply:
oh, you meant T=()...
this is OK, but requires https://github.com/paritytech/substrate/pull/14330 to avoid doubled methods on RuntimeGenesisConfig.

For me introducing new trait here is reasonable tradeoff.